### ISSUE TITLE
Text are hidden by the menu tool bar when you try to print posts on data view

### ISSUE THREAD

**Issue by CeciliaHinga** [ushahidi/platform#4141]
**Describe the bug**
When you try to print posts on data view, some of the content on the posts is hidden by the menu and 'bulk action' toolbars on the page preview 

**To Reproduce**
1. Go to data view
2. Click on fn+command+p for shortcut or file then print (This is with a mac laptop)
3. See error on the page preview

**Expected behavior**
Once you click on print the toolbars should not hide any text

**Is there a workaround? What is it.**
No workaround

**Screenshots**
<img width="475" alt="Screenshot_30_10_2020__11_33" src="https://user-images.githubusercontent.com/20906968/97692870-60c52b00-1ab1-11eb-824c-2dcde6418d2d.png">

**URL / Environment where this happened**
Production

**Desktop Hardware Details (please complete the following information):**
 - OS:  iOS
 - Browser chrome

Aha! Link: https://ushahiditeam.aha.io/features/PROD-890

**Comment by evanessa234** [comment]
@Angamanga can I look for solving this bug?

**Comment by Angamanga** [comment]
@evanessa234 Complete the issue you are working with before starting on the next one 🎉

**Comment by sunny775** [comment]
Hi @Angamanga 👋🏻 I am an Outreachy applicant, and I'd love to take on this issue. Can you assing me please

**Comment by Angamanga** [comment]
@sunny775 Welcome! Please read and follow the tasks here before starting on an issue: https://github.com/ushahidi/platform/discussions/4213 👍

**Comment by sunny775** [comment]
Thanks @Angamanga . I'll do that right away

**Comment by MugoBrian** [comment]
Hello @Angamanga. Since the screenshot is for the old platform client, is the issue relevant or does it occur in the new platform-client-mzima?

**Comment by prathamkandari** [comment]
@CeciliaHinga I would like to work on this issue. Can you please assign it to me ?

**Comment by Angamanga** [comment]
@MugoBrian Yes, it is still relevant for the new client! I updated the issue. Try printing the feed-view from the client and you will notice that the layout is off.

**Comment by Angamanga** [comment]
@prathamkandari I have added 2 issues that are open for everyone to start working on straight away, these are https://github.com/ushahidi/platform/issues/4749 and https://github.com/ushahidi/platform/issues/4750.

**Comment by MugoBrian** [comment]
Hello @Angamanga and @Ifycode, I am facing a blocker with this issue. 
Based on the print layout (see screenshot below), 

1. I cannot see which texts are hidden by the menu toolbar. 
2. **Desired Print Layout:** What should the resulting print layout look like since @Angamanga mentioned the layout looks off and how do I go about it?

PS: I have done some digging and even tried @media print to change the layout but it seems not to work. Do you have any pointers you could recommend?

**Comment by crocmons** [comment]
@MugoBrian this is for Mac users.. they gonna see this issue are you a Mac user or windows?

**Comment by MugoBrian** [comment]
I am using Ubuntu.

**Comment by Angamanga** [comment]
@MugoBrian I think you can look at this issue from the perspective of how a user would look at the page on a smaller screen, and use the same layout.
You can also think about it asking yourself "what is the most important information the user would want to print" and experiment with that. We unfortunately do not have any designs for it to provide.

**Comment by Angamanga** [comment]
@crocmons The issue is for everyone, the command in the description was just an example how the user who raised the issue triggered the print-page.

**Comment by Angamanga** [comment]
@MugoBrian Yes, `@media print {}` should be it. You can look at where we have other media-queries in the platform code 👍 Let me know if you need more info or hints on where to go :)

**Comment by MugoBrian** [comment]
Ok. I will inform you.
